### ISSUE TITLE
Default Interface Member Implementation Q.

### ISSUE THREAD

**Issue by RexJaeschke** [dotnet/docs#32871]
In recent weeks, I've been going over this proposal in order to write a formal spec for the C# Standard. At times, it has been heavy going!

The main obstacle was the (misleading) occurrence of the modifier `override` in numerous examples and narrative. As best as I can tell, this modifier is **not** actually permitted on any interface member. Instead, overriding is achieved via explicit implementation.

The remaining question has to do with the near-final topic "Base Interface Invocations," for which the decision "Decided on `base(N.I1<T>).M(s)`" was made. However, as far as I can tell, support for this was **not** added to the V8 compiler. Can someone please confirm (or refute) that.

BTW, I think the proposal is misnamed: it really applies to all interface function members (properties, indexers, and events as well), not just methods.

[Enter feedback here]

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 17012c83-0313-919b-eb8b-04fae8f4df4a
* Version Independent ID: 05252e5a-e53c-5ce5-84b3-671faad0549c
* Content: [Default interface methods - C# 8.0 draft specifications](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-8.0/default-interface-methods)
* Content Source: [proposals/csharp-8.0/default-interface-methods.md](https://github.com/dotnet/csharplang/blob/main/proposals/csharp-8.0/default-interface-methods.md)
* Product: **dotnet-csharp**
* Technology: **csharp-spec**
* GitHub Login: @dotnet-bot
* Microsoft Alias: **wiwagn**

**Comment by jcouv** [comment]
This would be good to discuss with Aleksey (implemented DIM)

**Comment by AlekseyTs** [comment]
> As best as I can tell, this modifier is **not** actually permitted on any interface member. Instead, overriding is achieved via explicit implementation.

That is correct. Also, I recommend to not use the term "overriding" anywhere in the specification in relation to DIM feature. Interface members cannot be overridden, they can only be implemented. In interfaces that can be achieved with explicit implementation syntax. In classes and structures, both implicit and explicit implementation can be used.

**Comment by AlekseyTs** [comment]
> The remaining question has to do with the near-final topic "Base Interface Invocations," for which the decision "Decided on `base(N.I1<T>).M(s)`" was made. However, as far as I can tell, support for this was **not** added to the V8 compiler. Can someone please confirm (or refute) that.

The "Base Interface Invocations" did not make it into the product

**Comment by BillWagner** [comment]
This has been fixed in the interim.
